File: Java Basic/Sum.java
//Take 2 variables 'a' and 'b' and print their sum.
import java.util.*;

public class Sum
{
    public static void main(String []args)
    {
        float a,b; // Use double also
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter 1st number: ");
        a=sc.nextFloat();
        System.out.print("Enter 2nd number: ");
        b=sc.nextFloat();
        System.out.println("Sum is "+(a+b));
        
        //Constanat value
        final int x=10;
        System.out.println(x);   
        sc.close();
    }
}

File: Ashish sir/Recursion/sum.cc
#include<iostream>
using namespace std;
int sum(int n)
{
    if(n==1) return 1;
    return n+sum(n-1);
}
int main()
{
    int n;
    cout<<"Enter number of disk: ";
    cin>>n;
    int x=sum(n);
    cout<<"Sum of number: "<<x;
    return 0;
}